src/data/bucket/symbol_bucket.js

```diff
@@ -539,7 +539,8 @@ class SymbolBucket implements Bucket {
                writingMode: any,
                labelAnchor: Anchor,
                lineStartIndex: number,
-               lineLength: number) {
+               lineLength: number,
+               associatedIconIndex: number) {
```

Comment:
Looking at the `addSymbols` calls, `associatedIconIndex` seems to be unused (always undefined). So it's added as `0`?

Reply:
No, there are two calls in `symbol_layout.js` both have an index added. Its fixed to `-1` for the icon bucket, and the `placedIconIndex` for the the text vertices in `addTextVertices`